syntax = "proto3";

package fib;

import "google/api/annotations.proto";

message FibRequest {
    string n = 1; 
}

message FibResponse {
    uint64 fib = 1;
    uint32 timestamp = 2;
    uint32 count = 3;
}

service FibService {
    rpc ComputeFib (FibRequest) returns (FibResponse) {
        option (google.api.http) = {
            get: "/fib"
        };
    }
}
